apps/frontend-manage/src/components/evaluation/TableChart.tsx

```diff
@@ -25,27 +25,23 @@ function TableChart({
 }: TableChartProps): React.ReactElement {
   const ref = useRef<{ reset: () => void }>(null)
 
+  const formatCorrect = (value: boolean | undefined | null): string => {
```

Comment:
I'm currently not aware of any other function where we need this formatter? But yes, as soon as we need it somewhere else as well, this would definitely be a good idea.